crates/aptos/src/create_account/mod.rs

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+//! A command to create a new account on-chain
+//!
+//! TODO: Examples
+//!
+
+use crate::{
+    common::types::{EncodingOptions, KeyInputOptions},
+    CliResult, Error as CommonError,
+};
+use anyhow::Error;
+use aptos_crypto::{ed25519::Ed25519PrivateKey, PrivateKey};
+use aptos_rest_client::Client as RestClient;
+use aptos_sdk::{
+    transaction_builder::TransactionFactory,
+    types::{chain_id::ChainId, transaction::authenticator::AuthenticationKey, LocalAccount},
+};
+use aptos_transaction_builder::aptos_stdlib;
+use aptos_types::account_address::AccountAddress;
+use clap::Parser;
+use reqwest;
+
+/// Command to create a new account on-chain
+///
+#[derive(Debug, Parser)]
+pub struct CreateAccount {
+    #[clap(flatten)]
+    key_input_options: KeyInputOptions,
+    #[clap(flatten)]
+    encoding_options: EncodingOptions,
+    /// Address of account you want to create
+    account: AccountAddress,
+    /// Chain ID
+    chain_id: u8,
+    #[clap(long, default_value = "https://fullnode.devnet.aptoslabs.com")]
+    node_url: String,
```

Comment:
We should make these a common type since they likely need to be used for any chain rest operation.  Then we can flatten them and have common documentation for all uses.